ci/jenkins/test.sh

```diff
@@ -621,6 +627,13 @@ function deliver_antrea {
             scp -q -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no -i "${WORKDIR}/jenkins_id_rsa" flow-aggregator.tar jenkins@[${IP}]:${DEFAULT_WORKDIR}/flow-aggregator.tar
             ssh -o StrictHostKeyChecking=no -i "${WORKDIR}/jenkins_id_rsa" -n jenkins@${IP} "${CLEAN_STALE_IMAGES}; docker load -i ${DEFAULT_WORKDIR}/antrea-ubuntu.tar; docker load -i ${DEFAULT_WORKDIR}/flow-aggregator.tar" || true
         done
+    elif [[ $TESTBED_TYPE == "kind" ]]; then
+         kubectl get nodes -o wide --no-headers=true | awk '{print $6}' | while read IP; do
+            docker load -i antrea-ubuntu.tar
+            kind load docker-image antrea/antrea-ubuntu:latest --name ${CLUSTER}
```

Comment:
If there are two jobs in parallel. Both jobs use antrea/antrea-ubuntu:latest , one will overwrite the image of another job. test.sh assumes that there is only one job on the testbed.

test-vmc.sh solves this issue. If we want to allow multiple jobs in parallel, we need to refactor test.sh to be like test-vmc.sh. When building the Antrea image, it should use different tags for each different job. After loading the image on the K8s node, it can tag the image back to :latest.

If this refactoring introduces too many changes, we can just accept this limitation. We can create more VMs as Jenkins nodes.